web/src/app/shared/components/list/base-list.directive.ts

```diff
@@ -159,7 +159,7 @@ export class BaseListDirective {
   socketListener(m: EventMessage): void {
     const stype = (x: string) => `${m.object.type}${m.object.details.type ? `2${m.object.details.type}` : ''}` === x;
 
-    const checkUpgradable = () => (m.event === 'create' || m.event === 'delete') && m.object.type === 'bundle' && this.typeName === 'cluster';
+    const checkUpgradable = () => (m.event === 'create' || m.event === 'delete') || (m.object.type === 'bundle' && this.typeName === 'cluster') || (m.object.type === 'service' && this.typeName === 'service2cluster');
```

Comment:
I think, first part (`(m.event === 'create' || m.event === 'delete')`) must necessarily exist in condition (first part && second part).
May be need change this function for more clarity: add list of upgradable events and check includes m.event into this list. And add list of upgradable pairs type and typename. What do you think?

Reply:
Yup, sorry, my bad.